kujira/assetlist.json

```diff
@@ -385,6 +385,572 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/local.png"
         }
       ]
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "ibc/E5CA126979E2FFB4C70C072F8094D07ECF27773B37623AD2BF7582AD0726F0F3",
+          "exponent": 0
+        },
+        {
+          "denom": "sol",
+          "exponent": 8
+        }
+      ],
+      "base": "ibc/E5CA126979E2FFB4C70C072F8094D07ECF27773B37623AD2BF7582AD0726F0F3",
+      "name": "Wormhole Solana",
+      "display": "sol",
+      "symbol": "whSOL",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "wormchain",
+            "base_denom": "factory/wormhole14ejqjyq8um4p3xfqj74yld5waqljf88fz25yxnma0cngspxe3les00fpjx/8sYgCzLRJC3J7qPn2bNbx6PiGcarhyx8rBhVaNnfvHCA",
+            "channel_id": "channel-9"
```

Comment:
We are also going to need the IBC connections registered. for this one, we need /_IBC/kujira-solana.json